apps/repo_jobs/test/integration/consumer_test.exs

```diff
@@ -16,10 +16,13 @@ defmodule RepoJobs.Integration.ConsumerTest do
 
   @moduletag :integration
   @queue "test.consumer.queue"
-
+  # TODO - whey are there two setup blocks ?
```

Comment:
to make the intent clear, the first setup block is for setting up rabbitmq and the second one for setting up the application we are going to test. there's nothing special with this